### DISCUSSION TITLE
Parameters for a dbx launch

### DISCUSSION THREAD

**Discussion by goldstein0101** [databrickslabs/dbx#604] [Q&A] [answered]
Hey, could u give me an example of --parameters argument for dbx launch, since parameters are used as an parameters of a task, not a workflow, and each workflow has single or many tasks, so how do i change paramaters of single task with dbx launch --parameters. Do u understand my point?

**Comment by renardeinside** [comment, marked as answer]
this doc should answer that - https://dbx.readthedocs.io/en/latest/guides/general/passing_parameters/
